patterns/multisig/VirtualMultisigRune.ts

```diff
@@ -44,41 +45,45 @@ export class VirtualMultisigRune<out C extends Chain, out U>
   encoded = this.value.map((m) => $virtualMultisig.encode(m))
   hex = this.encoded.map(hex.encode)
 
-  proxyBySenderAddr<X>(...[senderAddr]: RunicArgs<X, [Uint8Array]>) {
-    const signatories = this.inner.into(ValueRune).access("signatories")
-    return Rune
-      .tuple([signatories, this.proxies, senderAddr])
-      .map(([signatories, proxies, senderAddr]) =>
-        MultiAddress.Id(signatories[proxies[hex.encode(senderAddr)]!]!)
-      )
+  proxyBySenderAccountId<X>(...[senderAccountId]: RunicArgs<X, [senderAccountId: Uint8Array]>) {
+    const senderAccountIdHex = Rune.resolve(senderAccountId).map(hex.encode)
+    const senderProxyI = this.proxies.access(senderAccountIdHex)
```

Comment:
`senderProxyId` ?

Reply:
this is still missing unless you have yet to push it

Reply:
If the GH review interface is acting wonky, perhaps view in the branch's tree?

<img width="796" alt="Screen Shot 2023-03-24 at 1 06 22 PM" src="https://user-images.githubusercontent.com/4893548/227593372-59a12443-5834-4a07-8299-c3dc6f21c540.png">